consul/consul.go

```diff
@@ -0,0 +1,19 @@
+package consul
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/consul-k8s/version"
+	capi "github.com/hashicorp/consul/api"
+)
+
+// NewClient returns a Consul API client. It adds a required User-Agent
+// header that describes the version of consul-k8s making the call.
+func NewClient(config *capi.Config) (*capi.Client, error) {
```

Comment:
@ishustava what do you think about this new package location?

Reply:
lol. A joke that haunts @ndhanushkodi, @ishustava and @thisisnotashwin!

Reply:
Makes sense!